.travis.yml

```diff
@@ -64,7 +64,8 @@ script:
   - timeout $GREP_TIMEOUT grep -m 1 'Running migrations' <(docker logs --follow rapidpro 2>&1)
   - timeout $GREP_TIMEOUT grep -m 1 'Downloading geojson for relation_ids 192796' <(docker logs --follow rapidpro 2>&1)
   - timeout $GREP_TIMEOUT grep -m 1 'Imported geojson for relation_ids 192796' <(docker logs --follow rapidpro 2>&1)
-  - timeout $GREP_TIMEOUT grep -m 1 'spawned uWSGI http 1' <(docker logs --follow rapidpro 2>&1)
```

Comment:
I'd rather you keep this line and set the supervisor child processes to log to stdout/stderr as shown here: https://github.com/praekeltfoundation/docker-supervisor#usage

If we store logs in files nobody will ever see them because it's a container running somewhere that's hard to get to.